lib/github.sh

```diff
@@ -205,6 +203,21 @@ strap::github::api::token::create() {
     strap::abort "$msg"
   fi
 
+  #Test if we need to do an SSO enable
+  response="$(curl --silent --show-error -i -H "Authorization: token $token" -H 'Content-Type: application/json' https://api.github.com/user/issues)"
+  headers="$(echo "$response" | sed "/^\s*$(printf '\r')*$/q" | sed '/^[[:space:]]*$/d' | tail -n +2)"
+  if echo "$headers" | grep -q 'X-GitHub-SSO:'; then
+    # Linux based system
+    if command -v gio >/dev/null; then
+      gio open https://github.com/settings/tokens
+    # Mac
+    else
+      open https://github.com/settings/tokens
```

Comment:
This should have a `command -v open >/dev/null` check before calling this as well - there could be a linux OS distribution that doesn't have either `open` or `gio`.  If neither are found, a message should be printed telling the user to visit that URL in their browser and then exit strap.